docs/Compatibility.md

```diff
@@ -77,7 +77,7 @@ For a list of available integrations, and their configuration options, refer to
 | Excon                      | `excon`                    | `>= 0.50`                | `>= 0.50`                 | [Link][16]                          | [Link](https://github.com/excon/excon)                                       |
 | Faraday                    | `faraday`                  | `>= 0.14`                | `>= 0.14`                 | [Link][17]                          | [Link](https://github.com/lostisland/faraday)                                |
 | Grape                      | `grape`                    | `>= 1.0`                 | `>= 1.0`                  | [Link][18]                          | [Link](https://github.com/ruby-grape/grape)                                  |
-| GraphQL                    | `graphql`                  | `>= 1.7.9`               | `>= 1.7.9`                | [Link][19]                          | [Link](https://github.com/rmosolgo/graphql-ruby)                             |
+| GraphQL                    | `graphql`                  | `>= 2.2.6`               | `>= 2.2.6`                | [Link][19]                          | [Link](https://github.com/rmosolgo/graphql-ruby)                             |
```

Comment:
We could indicate the nuances of support:
`>= 2.2.6, >= `1.13.21` < `2.0`, >= `2.0.28` <`2.1`, >= `2.1.11` < `2.2``

But I think it would be simplest to just suggest the latest version like you already have listed. The extra detail about the backported support could be indicated in the release notes instead too.

Reply:
I agree here, given `ddtrace` 2.x series will officially support all 4 graphql versions (removing support woulb be a breaking change).